packages/core/src/pubsub/incoming-channel.ts

```diff
@@ -74,18 +68,20 @@ export class IncomingChannel extends Observable<IPFSPubsubMessage> {
     this.tasks = buildResubscribeQueue(logger);
   }
 
-  private async resubscribe(handler: (message: IPFSPubsubMessage) => void, complete: () => void, forceSubscribe = false): Promise<void> {
+  private async subscribeToIpfs(handler: (message: IPFSPubsubMessage) => void, complete: () => void): Promise<void> {
     const isRunning = this.ipfs && this.ipfs.pubsub && this.ipfs.pubsub.ls;
     if (isRunning) {
-      const listeningTopics = await this.ipfs.pubsub.ls();
-      const isSubscribed = listeningTopics.includes(this.topic);
-      if (!isSubscribed || forceSubscribe) {
-        await this.ipfs.pubsub.unsubscribe(this.topic, handler);
-        await this.ipfs.pubsub.subscribe(this.topic, handler);
-        const ipfsId = await this.ipfs.id();
-        const peerId = ipfsId.id;
-        this.pubsubLogger.log({ peer: peerId, event: 'subscribed', topic: this.topic });
+      const onError = (error) => {
+        console.warn(`IPFS pubsub subscription error, resubscribing: ${error}`)
```

Comment:
I have tried restarting the IPFS node while the Ceramic node is running, but I do not see this message :confused: